bootstrap/src/main/java/org/jboss/windup/bootstrap/commands/windup/RunWindupCommand.java

```diff
@@ -166,6 +170,16 @@ else if (Boolean.class.isAssignableFrom(option.getType()))
         if (!validationSuccess)
             return;
 
+        // In case of --unzippedAppInput, treat the directories in --input as unzipped applications.
+        // Otherwise, as a directory containing separate applications (default).
+        Boolean isExplodedApp = (Boolean) optionValues.get(ExplodedAppInputOption.NAME);
```

Comment:
Could you be more specific? I.e. if there's --sourceMode, we should imply --unzippedAppInput ?